.github/ISSUE_TEMPLATE/feature.yml

```
@@ -0,0 +1,249 @@
+        - [ ] User can successfully authenticate with Google OAuth
+        - [ ] JWT tokens are properly managed and refreshed
+        - [ ] Authentication state persists across app restarts
+        - [ ] All security tests pass
+        - [ ] Performance benchmarks are met (< 500ms)
+        - [ ] Accessibility audit passes (WCAG 2.1 AA)
+        - [ ] Works on iOS and Android
+        - [ ] All unit and widget tests pass
+        - [ ] Code review approval received
+      value: |
+        ### Core Functionality:
+        - [ ] 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)